packages/components/app/styles/components/pagination-bar.scss

```diff
@@ -110,21 +130,19 @@
 
 // * Sub-component: PaginationPageSize
 // ----------------------------------------
-// TODO: refactor? (Should not reference nested component classes if possible)
 .hds-pagination-page-size {
-  > * {
-    display: flex;
-    gap: 12px; // TODO: refactor
-    align-items: center;
-  }
+  display: flex;
+  grid-area: right;
+  gap: 12px; // TODO: refactor
+  align-items: center;
+  justify-self: flex-end;
 
-  // Issue: attributes are spread on nested select element, not on the component's parent wrapper...
-  .hds-form-label {
+  > label {
     font-weight: inherit;
     font-size: inherit;
```

Comment:
if you move the typographic classes to the `label` then you have to remove these two rules